rules/know-the-roles-Account-Manager-Scrum-Master-Tech-Lead/rule.md

```diff
@@ -0,0 +1,47 @@
+---
+type: rule
+archivedreason:
+title: Do you know the roles of the Account Manager, Scrum Master, and Tech Lead?
+guid: {guid}
```

Comment:
Hey @tiagov8, 

When you create a new rule, can you please replace `{guid}` with a GUID from https://www.guidgen.com

I have already done this for this rule [here](https://github.com/SSWConsulting/SSW.Rules.Content/pull/678/commits/41d2c491fa1ffdcc2fe333d6de251bb2c5d1fef9)

Thanks,

-Brady